core/models2/models.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
```

Comment:
Is this file a remnant? Let's do 1 model = 1 file like you've done elsewhere (it'll be cleaner as we build this out)